pipeline/sarek.config

```diff
@@ -18,4 +18,5 @@ profiles {
   munin { includeConfig "${params.custom_config_base}/conf/pipeline/sarek/munin.config" }
   sbc_sharc { includeConfig "${params.custom_config_base}/conf/pipeline/sarek/sbc_sharc.config" }
   uppmax { includeConfig "${params.custom_config_base}/conf/pipeline/sarek/uppmax.config" }
+  azurebatch { includeConfig "${params.custom_config_base}/conf/pipeline/rnaseq/azurebatch_pools_Ddv4.config" }
```

Comment:
Can we have that at the top of the list?
I'm sorry, I just love sorted lists

Reply:
Done